src/components/Logo/Logo.story.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import Logo from './Logo';
+
+const stories = storiesOf('Branding', module);
+
+stories.add('Logo', () => (
+  <Logo
+    kind={select('Kind', ['logo mark', 'full'], 'full')}
+  />
+  ),
+  {
+    info: {
+      text: `Fluid logo that scales up to a width of the small viewport 
```

Comment:
give the width in units to be more explicit, then again this could also be solved by the story source tabs implementation

Reply:
These change depending on breakpoint though. It could easily become inconsistent.

Reply:
valid point, since they are actually based on rems they are based off of the body font size which is fluid with viewport width